msteams-platform/sbs-tabs-and-messaging-extensions-with-sso.yml

```diff
@@ -421,79 +421,81 @@ items:
+       
       1. If SSO isn't performed, then the bot will perform default authentication method for your tab.
 
-          ![No SSO](~/assets/images/Tab-ME-SSO/not-login11.png)
-
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/not-login11.png" alt-text="Screenshot shows the default authentication method.":::
+          
       1. Select ellipses **...** to open message extension.
 
-          ![message extension](~/assets/images/Tab-ME-SSO/message-extension.png)
-      
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/message-extension.png" alt-text="Screenshot shows the message extension.":::
+                
       1. Select your new SSO application.
 
-          ![SSO app](~/assets/images/Tab-ME-SSO/recent-app-sso.png)
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/recent-app-sso.png" alt-text="Screenshot shows the SSO app.":::
 
       1. Select **sign in**.
 
-          ![Sign in](~/assets/images/Tab-ME-SSO/sign-in-sso1.png)
-
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/sign-in-sso1.png" alt-text="Screenshot shows the Sign in page.":::
+          
       1. Enter your credentials in sign-in box, and select **Next**.
 
-          ![Sign in box](~/assets/images/Tab-ME-SSO/sign-in-box.png)
-      
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/sign-in-box.png" alt-text="Screenshot shows the Sign in box to enter the credentials.":::
+                
       1. Select **Accept**.
 
-          ![consent box](~/assets/images/Tab-ME-SSO/concent-box.png)
-
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/concent-box.png" alt-text="Screenshot shows the consent box.":::
+          
       1. Open message extension and select your application.
 
       1. Select your profile under search box. You'll see something like this:
 
-          ![Profile](~/assets/images/Tab-ME-SSO/hello-megan-profile.png)
-
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/hello-megan-profile.png" alt-text="Screenshot shows the profile.":::
+          
       1. Send `https://profile.botframework.com` to the application to get a profile card.
        
-          ![Profile card](~/assets/images/Tab-ME-SSO/profile-card.png)
-
+          :::image type="content" source="~/assets/images/Tab-ME-SSO/profile-card.png" alt-text="Screenshot shows the profile card.":::
+          
 - title: Complete challenge
   durationInMinutes: 1
   content: |
     Did you come up with something like this?
 
-       ![SSO profile](~/assets/images/Tab-ME-SSO/hello-megan-profile245.png)
-    
+       :::image type="content" source="~/assets/images/Tab-ME-SSO/hello-megan-profile245.png" alt-text="Screenshot shows the SSO profile.":::
+       
     
-
 - content: |
+Doc-Improvement-Updating-alt-test-for-images9
```

Comment:
Removing branch name from within content `Doc-Improvement-Updating-alt-test-for-images9`.